pkg/kubelet/apis/config/types.go

```diff
@@ -397,6 +397,24 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdown
 	// +optional
 	ShutdownGracePeriodCriticalPods metav1.Duration
+	// podPriorityShutdownGracePeriods specifies the shutdown grace period for Pods based on their associated priority class value. Requires the GracefulNodeShutdown feature gate to be enabled.
```

Comment:
Expect the comment for this (or `ShutdownGracePeriod`) to talk about the interaction between it and `ShutdownGracePeriod` (i.e. "must be less than ShutdownGracePeriod").